tests/unit/test_exception_handler.py

```diff
@@ -23,11 +23,9 @@
 import logging
 import traceback
 import unittest
-
 import mock
 
-from hpOneView.exception_handler import handle_exceptions
-from hpOneView.exceptions import HPOneViewException
+from hpOneView.exceptions import HPOneViewException, handle_exceptions
 
 
 class ExceptionHandlerTest(unittest.TestCase):
```

Comment:
I believe this test class should be moved to the tests/unit/test_exception.py file.